### PR TITLE
add special operator count()

### PR DESCRIPTION
**Pull request by staltz**
The benchmarks for this look great!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -464,11 +465,15 @@ some after processing that you wouldn't create and index for, but the
 overhead of decoding the buffers is small enough that I don't think it
 makes sense.
 
-### all(operation, seq, descending, onlyOffset, cb)
+### all(operation, seq, limit, descending, onlyOffset, cb)
```
Comment (arj03) [inline README.md:468]: Should not have limit here

**Comment by arj03** [comment]
Yeah, was expecting some good numbers there :)

File: README.md (hunk 1 above)
Comment (staltz) [reply README.md:468]: Lol, sorry

**Comment by arj03** [comment]
The PR looks good. Just need the updated debug in.

**Review by arj03 — APPROVED** [review]

**Comment by staltz** [comment]
Done :)

**Pull request merged**